modules/tinyexr/config.py

```diff
@@ -3,4 +3,15 @@ def can_build(env, platform):
 
 
 def configure(env):
-    pass
+    from SCons.Script import BoolVariable, Variables, Help
+
+    envvars = Variables()
+    envvars.Add(
+        BoolVariable(
+            "tinyexr_export_templates",
+            "Enable saving and loading OpenEXR images in export template builds (increases binary size)",
+            False,
+        )
+    )
+    envvars.Update(env)
+    Help(envvars.GenerateHelpText(env))
```

Comment:
This isn't the correct way to do it, see https://github.com/akien-mga/godot/commit/e75da97e3b70ec83aebd0ccaaa6e363700901635 for an example.